java/src/main/java/org/whispersystems/signalservice/loki/api/LokiPublicChatAPI.kt

```diff
@@ -210,6 +210,19 @@ class LokiPublicChatAPI(private val userHexEncodedPublicKey: String, private val
         }
     }
 
+    public fun deleteMessages(messageServerIDs: List<Long>, channel: Long, server: String, isSentByUser: Boolean): Promise<List<Long>, Exception> {
+        return retryIfNeeded(maxRetryCount) {
+            val isModerationRequest = !isSentByUser
+            val parameters = mapOf( "ids" to messageServerIDs.joinToString() )
```

Comment:
The default separator used by `joinToString()` is `, ` (with a space). Those spaces are probably trimmed out but still it'd be nicer to explicitly specify `,` so that we don't end up with spaces in the query parameters